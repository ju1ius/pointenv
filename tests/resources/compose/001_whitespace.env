
  WS_AROUND  =  ws is stripped   
S_MULTILINE='foo
bar'
D_MULTILINE="foo
bar"

SAME_LINE="one" SAME_LINE_2='two' SAME_LINE_3=three SAME_LINE_4='four'
