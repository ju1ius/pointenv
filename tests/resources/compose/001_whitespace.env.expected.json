[
  {
    "key": "D_MULTILINE",
    "value": "foo\nbar"
  },
  {
    "key": "SAME_LINE",
    "value": "one"
  },
  {
    "key": "SAME_LINE_2",
    "value": "two"
  },
  {
    "key": "SAME_LINE_3",
    "value": "three SAME_LINE_4='four'"
  },
  {
    "key": "S_MULTILINE",
    "value": "foo\nbar"
  },
  {
    "key": "WS_AROUND",
    "value": "ws is stripped"
  }
]
